packages/babel-runtime-corejs2/helpers/esm/iterableToArray.js

```diff
@@ -1,5 +1,9 @@
 import _Array$from from "../../core-js/array/from";
+import _Set from "../../core-js/set";
+import _Map from "../../core-js/map";
 import _isIterable from "../../core-js/is-iterable";
+import _Symbol from "../../core-js/symbol";
```

Comment:
This also shouldn't need to be pulled in; it's referring to the global.

Reply:
It looks like checking the `typeof Symbol === 'function'` results in the build bringing in import, however, the use of `Symbol.iterator in Object(iter)` relies on the global. Do we want to explicitly rely on the global in the type check? i.e. `typeof global.Symbol === 'function'`

Reply:
`Symbol.iterator in something` is a special case which transpiles to `isIterable` helper.